recipes/opencv/4.x/conanfile.py

```diff
@@ -237,13 +242,29 @@ def _configure_cmake(self):
         if self.options.parallel:
             self._cmake.definitions["WITH_TBB"] = self.options.parallel == "tbb"
             self._cmake.definitions["WITH_OPENMP"] = self.options.parallel == "openmp"
+            self._cmake.definitions["WITH_PTHREADS_PF"] = self.options.parallel == "pthreads"
 
         self._cmake.definitions["ENABLE_PIC"] = self.options.get_safe("fPIC", True)
 
         if self.settings.compiler == "Visual Studio":
             self._cmake.definitions["BUILD_WITH_STATIC_CRT"] = "MT" in str(self.settings.compiler.runtime)
-        self._cmake.configure(build_folder=self._build_subfolder)
 
+        if self.settings.os == "Android":
+            self._cmake.definitions["ANDROID_STL"] = "c++_static"
+            self._cmake.definitions["ANDROID_NATIVE_API_LEVEL"] = self.settings.os.api_level
+            self._cmake.definitions["ANDROID_ABI"] = tools.to_android_abi(str(self.settings.arch))
+            self._cmake.definitions["BUILD_ANDROID_EXAMPLES"] = False
+            # TODO: "Use NVidia carotene acceleration library for ARM platform"
+            # if we want to support this, we must include an extra library 
+            # "lib/libtegra_hal.a" for shared=False, don"t know how to do that.
+            # Therefore we disable this for the moment
+            self._cmake.definitions["WITH_CAROTENE"] = False
+            if "ANDROID_NDK_HOME" in os.environ:
+                self._cmake.definitions["ANDROID_NDK"] = os.environ.get("ANDROID_NDK_HOME")
+
+        # TODO: avoid python2 warning and numpy error, is this the best way to do it?
+        self._cmake.definitions["OPENCV_PYTHON_SKIP_DETECTION"] = True
```

Comment:
This was not required previously 🤔

Reply:
Right, I will remove it. I thought the messages generated without this are the reasons for many failing fests, but I guess it is not.